### ISSUE TITLE
Can you share the pre-train weight of IGCV3 on the ImageNet?

### ISSUE THREAD

**Issue by irfanICMLL** [xxradon/IGCV3-pytorch#1]
Hi,
Great work!
Can you share the pre-train weight of IGCV3 on the ImageNet?
I am working on doing some segmentation tasks with the backbone of IGCV3. Can you share your pre-train weight?

Yours,

Yifan

**Comment by xxradon** [comment]
I'm so sorry,I do not have enough GPUs to train IGCV3 on ImageNet.If you have resources,you can train it and let me know the result.

**Comment by irfanICMLL** [comment]
So the results are from the paper?

**Comment by xxradon** [comment]
No,I have trained the IGCV3 on cifar 10 and cifar 100,the result is listed on README.md

**Comment by Re3write** [comment]
hi，i got  error when i insert the permutation block in my code,and my code just look like this ,do u have any good idea?

![image](https://user-images.githubusercontent.com/41424260/49276423-768ed780-f4b9-11e8-934f-a3c99d6ea273.png)



class Inverted_Bottleneck(nn.Module):
    expansion = 4

    def __init__(self, inplanes, planes, stride=1, downsample=None,
                 residual=True):
        super(Inverted_Bottleneck, self).__init__()
        self.conv1 = nn.Conv2d(inplanes, inplanes*6, kernel_size=1, bias=False,groups=2)
        self.bn1 = BatchNorm(inplanes*6)
        self.permutation1=PermutationBlock(groups=2),
        self.conv2 = nn.Conv2d(inplanes*6, inplanes*6, kernel_size=3, stride=stride,bias=False,
                              groups=inplanes*6)
        self.bn2 = BatchNorm(inplanes*6)
        self.conv3 = nn.Conv2d(inplanes*6, planes * 4, kernel_size=1, bias=False,groups=2)
        self.bn3 = BatchNorm(planes * 4)
        self.relu = nn.ReLU(inplace=True)
        self.permutation2=PermutationBlock(groups=int(round((planes*4 / 2)))),
        self.downsample = downsample
        self.stride = stride
        # self.se = se_block.SELayer(planes * 4)

    def forward(self, x):
        residual = x

        out = self.conv1(x)
        out = self.bn1(out)
        out = self.relu(out)

        out=self.permutation1(out)

        out = self.conv2(out)
        out = self.bn2(out)
        out = self.relu(out)

        out = self.conv3(out)
        out = self.bn3(out)
        # out = self.se(out)
        out=self.permutation2(out)

        if self.downsample is not None:
            residual = self.downsample(x)

        out += residual
        out = self.relu(out)

        return out
